deployment/commons/sso/keycloak/entrypoint/run-keycloak

```diff
@@ -15,22 +15,25 @@
 
 set -e
 
+KEYCLOAK_CERT_FILE="etc/x509/https/${SSO_CERT_FILE}"
+KEYCLOAK_KEY_FILE="etc/x509/https/${SSO_KEY_FILE}"
+
 if [ ! -f /opt/keycloak/first-run ]; then
   REALM_NAME="kapua"
-  KC_PORT="9090"
-  KC_URL="http://localhost:$KC_PORT"
+  KC_URL="http://localhost:$KEYCLOAK_PORT_HTTP"
   KC=/opt/keycloak/bin/kcadm.sh
 
   echo "Kapua Keycloak Configuration:"
   echo "    Kapua Console URL: $KAPUA_CONSOLE_URL"
   echo "    Keycloak Realm: $REALM_NAME"
   echo "    Keycloak Admin User: $KEYCLOAK_USER"
+  echo "    Keycloak TLS disabled:  $KEYCLOAK_DISABLE_SSL"
```

Comment:
Please don't use this reverse logic ENV to enable component. 

It is far easier to understand `KEYCLOAK_SSL_ENABLE` when is defined and used. 
The default can be `false` to indicate that a feature is disabled by default.